contrib/gitian-build.sh

```diff
@@ -1,453 +1,93 @@
-#!/usr/bin/env bash
-# Copyright (c) 2016 The Bitcoin Core developers
-# Distributed under the MIT software license, see the accompanying
-# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+#!/bin/bash
```

Comment:
Where does this come from? We are replacing bitcoin version with something entirely else. Why?

Reply:
This is the script we are using now for releases.

Reply:
Reverted as it's better to leave it as is.

Reply:
Ok. I think that if you want to add a script you can always add a new file without re-purposing the existing one. Also bitcoin changed shell script here to python in latest versions